### PR TITLE
Support Boost 1.70+

### PR DESCRIPTION
**Pull request by jmarrec**
Pending a new release of https://github.com/zaphoyd/websocketpp, add patch diffing between current websocket master and develop, mostly to capture https://github.com/zaphoyd/websocketpp/pull/814 that enables support for boost 1.70+

### CODE REVIEW

File: conanfile.py [hunk 1]
```diff
@@ -23,15 +23,19 @@ def requirements(self):
...
-            self.requires.add('boost/1.69.0@conan/stable')
+            self.requires.add('boost/1.71.0')
 
     def source(self):
         archive_name = "{0}-{1}".format(self.name, self.version)
         tools.get("{0}/archive/{1}.tar.gz".format(self.homepage, self.version),
                   sha256="178899de48c02853b55b1ea8681599641cedcdfce59e56beaff3dd0874bf0286")
         os.rename(archive_name, self._source_subfolder)
 
+        # Patch for boost 1.70+ support
```
Comment (Croydon) [inline conanfile.py:34]: Could you please directly mention here, that this patch won't be necessary anymore with the release after 0.8.1?

**Comment by Croydon** [comment]
By the way, @uilianries is adding Boost 1.68 to CCI now assumingly because of websocketpp https://github.com/conan-io/conan-center-index/pull/498

File: conanfile.py (hunk 1 above)
Comment (jmarrec) [reply conanfile.py:34]: Done

File: conanfile.py [hunk 2]
```diff
@@ -23,15 +23,20 @@ def requirements(self):
...
+        # TODO: will not be necessary anymore with websocket release > 0.8.1
```
Comment (jmarrec) [inline conanfile.py:35]: @Croydon

**Comment by jmarrec** [comment]
Thanks for the review @Croydon. Do you happen to know when the next testing => stable merge will happen?

**Comment by Croydon** [comment]
Whenever we see it fit 😄 

Done

**Pull request merged**